reg.js

```diff
@@ -56,7 +56,54 @@ $(document).ready(function(){
 				document.getElementById('register_btn').disabled = false;
 			}
     	});
-   	})
+		 })
+	$('#team_code_btn').click(function () {
+		document.getElementById("team_username_input").disabled = true;
+		document.getElementById("team_code_btn").disabled = true;
+
+		username = document.getElementById('team_username_input').value.toLowerCase();
+		if (username == '') {
+			alert('Username cannot be blank.');
+			return;
+		}
+		$.ajax({
+			url: "reg.php?func=get_code&username=" + username,
+			success: function (response) {
+				if (response == 'username invalid') {
+					alert('Username invalid. Please try again.');
+					document.getElementById("team_username_input").disabled = false;
+					document.getElementById("team_code_btn").disabled = false;
+					return;
+				}
+				document.getElementById('team_code').innerHTML = response;
+				document.getElementById('instructions_div').style.display = 'block';
```

Comment:
Why is this being dynamically set?

Reply:
We don't know the code until it gets generated by the PHP server. `team_code` refers to the code that is used to verify the user because they're supposed to put it in the Location field of their profile on MAL.

Reply:
I meant the block styling for #instructions_div

Reply:
Oh it's because in index.php it's defined as `<div id='instructions_div' style='display:none'>` so you can't see it until the Continue button is clicked below the username input, where the style is set to block styling.

The registration code also does the same thing to show the dropdown boxes after you enter your username and click Continue, so I just followed that technique.

Reply:
Gotcha, thanks.